coworking_agreement.md

```diff
@@ -1,25 +1,44 @@
-# Coworking Agreement
+.# Coworking Agreement
```

Comment:
Thanks for filling this out ❤️ Even if what you listed out mostly stuff you're already doing, it can be really helpful to keep in mind and help make sure folks aren't relying on assumptions about how they will work together.